Source/relay/TourGuide/Items of the Month/2021/Potted Power Plant.ash

```diff
@@ -2,52 +2,51 @@
 RegisterResourceGenerationFunction("IOTMPowerPlantGenerateResource");
 void IOTMPowerPlantGenerateResource(ChecklistEntry [int] resource_entries)
 {
-	if (!lookupItem("potted power plant").have())
-		return;
-	// Title
-	string [int] description;
-	string batteriesToHarvest = (get_property("_pottedPowerPlant"));
-	// Entries
-	if (batteriesToHarvest != "0,0,0,0,0,0,0")
-	{
-		string main_title = "Power plant batteries";
-		string [int] harvest;
-		harvest.listAppend("Harvest your potted power plant batteries.");
-		resource_entries.listAppend(ChecklistEntryMake("__item potted power plant", "inv_use.php?pwd=" + my_hash() + "&whichitem=10738", ChecklistSubentryMake(main_title, "", harvest), 1));
-	}
+  if (!lookupItem("potted power plant").have())
```

Comment:
thank you for fixing these tabs. at some point we should set up a prettier linter for this repo.